### PR TITLE
Changelog 129

### PR DESCRIPTION
**Pull request by mchernyavsky**

### CODE REVIEW

File: _posts/2020-08-24-changelog-129.markdown [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5873 -->
+* Improve autocompletion of the `cargo` command in the [Run Anything](https://www.jetbrains.com/help/clion/rust-support.html#cargo-support) popup
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5641 -->
+* Don't annotate user-defined attribute macros that override built-in ones (by [@Stzx])
+
+## Internal Improvements
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5932 -->
+* Initial regression testing
```
Comment (Undin) [inline _posts/2020-08-24-changelog-129.markdown:83]: Let's add something like `It should help to catch false-positive errors in new changes automatically`

File: _posts/2020-08-24-changelog-129.markdown [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+layout: post
+title: "IntelliJ Rust Changelog #129"
+date: 2020-08-24 04:01:13 +0300
+---
+
+
+## New Features
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5745 -->
+* Implement `wasm-pack` run configuration
```
Comment (Undin) [inline _posts/2020-08-24-changelog-129.markdown:11]: Maybe add link (https://rustwasm.github.io/docs/wasm-pack/) to wasm-pack?

**Review by Undin — CHANGES_REQUESTED** [review]

File: _posts/2020-08-24-changelog-129.markdown [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+title: "IntelliJ Rust Changelog #129"
+date: 2020-08-24 04:01:13 +0300
+---
+
+
+## New Features
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5745 -->
+* Implement [`wasm-pack`](https://rustwasm.github.io/docs/wasm-pack/) run configuration.
+  So now you can build your WASM library using `wasm-pack` just from the IDE
```
Comment (Undin) [inline _posts/2020-08-24-changelog-129.markdown:12]: Hm, actually, it's not only about build. Maybe it's better to write something like "So now you can run any `wasm-pack` command just from the IDE"? What do you think?
Sorry for confusing first comment

File: _posts/2020-08-24-changelog-129.markdown [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+* Suggest the run configuration name from its command
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5645 -->
+* Support the `Complete Current Statement` action for function and structs (by [@Stzx])
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5828 -->
+* Allow folding of consecutive `impl Iterator<Item=X>`s in a method chain hints (by [@Kobzol])
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5760 -->
+* Show error message when trying to run with coverage on non-nightly channel
```
[the remark this answers is not in the document]
Comment (Undin) [reply _posts/2020-08-24-changelog-129.markdown:55]: Did you skip this comment by accident or intentionally?

File: _posts/2020-08-24-changelog-129.markdown [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+    <!-- https://github.com/intellij-rust/intellij-rust/pull/5910 -->
+    * Create default run configuration for a new project
+
+    <!-- https://github.com/intellij-rust/intellij-rust/pull/5919 -->
+    * Improve package name validation during the new project creation
+
+* Lints & inspections improvements:
+
+    <!-- https://github.com/intellij-rust/intellij-rust/pull/5889 -->
+    * Move all `rustc`-related lints to the newly created separate "Lint" group.
```
Comment (Undin) [inline _posts/2020-08-24-changelog-129.markdown:30]:
```suggestion
    * Move all `rustc`-related lints to the newly created separate `Lints` group.
```

File: _posts/2020-08-24-changelog-129.markdown [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5745 -->
+* Implement [`wasm-pack`](https://rustwasm.github.io/docs/wasm-pack/) run configuration.
+  So now you can build your WASM library using `wasm-pack` just from the IDE
+
+* Project creation improvements:
+
+    <!-- https://github.com/intellij-rust/intellij-rust/pull/5845 -->
+    * Provide support for custom project templates in the `New Project` dialog
+    
+        <img src="/assets/posts/changelog-129/cargo-generate.png" alt="custom user template screenshot" width="646px"/>
```
Comment (avrong) [inline _posts/2020-08-24-changelog-129.markdown:19]: I don't think that this template would work using cargo-generate. Currently, cargo-generate provides support only for [its special templates](https://github.com/ashleygwilliams/cargo-generate/blob/master/TEMPLATES.md).

As an option, here is updated screenshot.

**Comment by mchernyavsky** [comment]
@Undin Fixed.

File: _posts/2020-08-24-changelog-129.markdown [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5940 -->
+* Suggest the run configuration name from its command
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5828 -->
+* Allow folding of consecutive `impl Iterator<Item=X>`s in a method chain hints (by [@Kobzol])
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5760 -->
+* Don't force to use the `nightly` toolchain when run with coverage.
+  Instead, IDE will now show the error message complaining about using a non-nightly channel.
+  So now user can use any nightly toolchain.
```
Comment (Undin) [inline _posts/2020-08-24-changelog-129.markdown:57]: Let's split it into two items:

```suggestion
<!-- https://github.com/intellij-rust/intellij-rust/pull/5760 -->
* Code coverage integration improvements:
    * Allow using any nightly toolchain to run code with coverage.
    * Show the error message if a user tries to launch code with coverage without nightly toolchain
```

**Pull request merged**